config/initializers/sentry.rb

```diff
@@ -0,0 +1,42 @@
+require "sentry-raven"
+
+Raven.configure do |config|
+  config.dsn = ENV["SENTRY_DSN"]
+  config.current_environment = Rails.env.to_s
```

Comment:
```suggestion
  config.current_environment = ENV['SENTRY_CDX_ENVIRONMENT'] || Rails.env.to_s
```

Let's allow to tell different productive environments apart if needed.